src/commands.rs

```diff
@@ -3967,6 +3977,20 @@ fn cmd_git_remote_remove(
     Ok(())
 }
 
+fn cmd_git_remote_rename(
+    ui: &mut Ui,
+    command: &CommandHelper,
+    args: &GitRemoteRenameArgs,
+) -> Result<(), CommandError> {
+    let workspace_command = command.workspace_helper(ui)?;
+    let repo = workspace_command.repo();
+    let git_repo = get_git_repo(repo.store())?;
+    git_repo
+        .remote_rename(&args.old, &args.new)
+        .map_err(|err| CommandError::UserError(err.to_string()))?;
```

Comment:
`jj git remote remove` manually checks for the remote, but the operation already contains that check and produces a pleasant enough error: `Error: remote 'asdf' does not exist; class=Config (7); code=NotFound (-3)`. Are we intentionally replacing that error with our own, perhaps to exclude the weird trailing details?

Reply:
Makes sense to me!